dataplicity/m2m/commandservice.py

```diff
@@ -38,54 +38,83 @@ def __init__(self, channel, command):
     def __repr__(self):
         return self._repr
 
+    @classmethod
+    def send_error(cls, channel, status, msg, **extra):
```

Comment:
> Interesting... I don't think we should be doing strange things to the code to make codacy happy. How did it complain? I suppose it said something about it not using self? In that case, perhaps it would make more sense to make this a function instead. That way, the duplication between the two modules here might be easier to resolve too.

It said it could be a static method. I went with classmethod because they are easier to extend if inheritance is involved. 

I fully agree re codacity. But this change isn't all that strange, since it doesn't need `self`. I think it is a better encapsulation to keep it in the class rather than a function.

Should we extend the services concept, and I suspect we will, I'd probably implement a base class, which would resolve the overlap of send_error.